src/server/common/connectors/file-upload/cdp-uploader.test.js

```diff
@@ -0,0 +1,145 @@
+import { checkStatus, uploadFile } from './cdp-uploader.js'
+import Wreck from '@hapi/wreck'
+import { config } from '~/src/config/config.js'
+import { spyOnConfig } from '../../test-helpers/config.js'
+
+/**
+ * @import { IncomingMessage } from 'node:http'
+ */
+
+jest.mock('@hapi/wreck')
+
+const maxFileSize = 10485760
+
+const mockUploaderUrl = 'http://mock-uploader-url'
+const mockBucket = 'mock-bucket'
+const mockPath = 'mock-path'
+const mockRedirectUrl = 'http://mock-redirect-url'
+const mockUploadId = 'test-upload-id'
+const mockUploadUrl = 'test-upload-url'
+const mockStatusUrl = 'test-status-url'
+
+const mockUploadConfig = {
+  bucket: mockBucket,
+  uploaderUrl: mockUploaderUrl,
+  path: mockPath
+}
+
+describe('uploadFile', () => {
+  beforeEach(() => {
+    config.get = jest.fn().mockImplementation((key) => {
+      if (key === 'fileUpload') {
+        return mockUploadConfig
+      }
+      if (key === 'featureFlags') {
+        return { pdfUpload: false }
+      }
+    })
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
```

Comment:
`jest.restoreAllMocks()` will make sure that we clear down the above spy correctly: https://jestjs.io/docs/jest-object#jestrestoreallmocks